### DISCUSSION TITLE
Dashboard page CSS issue

### DISCUSSION THREAD

**Discussion by kursatsmsek** [community/community#55040] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

Hi guys,

I'm sure about your working a lot to giving us a good experience and I guess you forgot something about Github.com dashboard CSS media. I wanted to let you know even the issue is a small cause it leads to a big lack of appearance.

I'm sending necessary photos as attach file here. 

Best of luck with your work.

<img width="1280" alt="Screen Shot 2023-05-10 at 13 32 27" src="https://github.com/community/community/assets/80540635/4ea5ef5c-ca00-41ce-b4c6-bde802793d96">
<img width="1280" alt="Screen Shot 2023-05-10 at 13 32 45" src="https://github.com/community/community/assets/80540635/d1e82d9c-425f-4594-85ef-e360265d8d72">

Here is solution that I think.
<img width="628" alt="Screen Shot 2023-05-10 at 13 33 33" src="https://github.com/community/community/assets/80540635/a0fb2b7a-cb30-4691-acef-e384f91bc9d6">

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @kursatsmsek ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
